### ISSUE TITLE
Карта не видна

### ISSUE THREAD

**Issue by casper1149** [tulov/angular-yandex-map#41]
Карта не отображается, в отличии от vanilla yandex карт, пример
http://plnkr.co/edit/Wvf7EPUprFCqNTCDkMS0

**Comment by tulov** [comment]
Эта проблема появится, если использовать angular версии больше чем 1.1.5. По мере наличия свободного времени разберусь.

**Comment by tulov** [comment]
Попробуйте с файлом в ya-map-2.2.js в папке example/2.2. У меня все работает.
Если проблем не возникнет, тогда обновлю в основном репозитории.

**Comment by casper1149** [comment]
Проверил, работает, спасибо за оперативность.
Единственное, заметил, что в папке с примерами, в одном из примеров используется 
ng-switch-when="true" с этой директивой вылезает ошибка
Controller 'ngSwitch', required by directive 'ngSwitchWhen', can't be found!
без нее все работает

**Comment by tulov** [comment]
Спасибо за участие. Посмотрю по мере возможности. Желательно укажите в каком примере.

**Comment by casper1149** [comment]
Пример директивы взял из example/2.2/partials/create.html.
Проблема в том, что в plnkr сходу воспроизвести не удалось, когда подключен модуль карт и только angular 1.4.2, в моем проекте подключено больше, например angular-animate и прочее, видимо где-то конфликт, будет время постараюсь воспроизвести.

**Comment by casper1149** [comment]
Упс, в Chrome карты не видна, в консоле выводится:
mapInit [37.64, 55.76]
Uncaught TypeError: Browser.addMessageHandlers is not a function

**Comment by casper1149** [comment]
Проблема видимо связана с тем, что я отображаю карту внутри ui-view, если кладу директиву на главный view то оно отображается, буду разбираться.

**Comment by casper1149** [comment]
Даже не так, все оказалось проще, проблема связана с размерами контейнера

**Comment by LennyLip** [comment]
Что-то у меня сегодня тоже перестала открываться. (использовал 2.1) одна перестала, вторая работала (вторая была с меньшим набором атрибутов)
Поменял на 2.2. пока пусто. (mapinit в консоли), раньше работало без размеров, теперь надо размеры прописывать?

**Comment by LennyLip** [comment]
хм, да, 
   <div style="height: 300px">
        <ya-map />
   </div>

помогло, ya-map - 0px по умолчанию.

**Comment by umed48** [comment]
как-то непонятно, как задавать размеры карты? пример в readme не помешал бы

**Comment by umed48** [comment]
Разобрался, необходимо для `ya-map` выставить высоту и `display: block;`. Но момент не очевиден.

**Comment by fortuneteller** [comment]
Да, про `display:block` неплохо было бы написать =). Съело много времени. Видимо из-за flex верстки, в которую вставлялась директива, игнорировались размеры контейнера (пока в своем css не прописал `  ya-map {
    display: inline-block;
  }
`). Как вариант можно сделать карту не непосредственно внутри `<ya-map>`, а добавить внутрь еще div с уже указанным `display:block`, и в него уже поместить карту. Тогда у всех должно работать из коробки, без танцев.

**Comment by m21448** [comment]
1.5 часа искал что нужно добавить style="display: block".
